notebooks/amazon_forest_notebook.py

```diff
@@ -201,39 +198,72 @@
 # <markdowncell>
 
 # ## Create a checkpoint
+# 
+# Creating a checkpoint saves the best model weights across all epochs in the training process. This ensures that we will always use only the best weights when making our predictions on the test set rather than using the default which takes the final score from the last epoch. 
+
+# <codecell>
+
+from tensorflow.contrib.keras.api.keras.callbacks import ModelCheckpoint
+
+filepath="weights.best.hdf5"
+checkpoint = ModelCheckpoint(filepath, monitor='val_acc', verbose=1, save_best_only=True)
 
 # <markdowncell>
 
-# Creating a checkpoint saves the best model weights across all epochs in the training process. This ensures that we will always use only the best weights when making our predictions on the test set rather than using the default which takes the final score from the last epoch. 
+# ## Choose Hyperparameters
+# 
+# Choose your hyperparameters below for training. 
+# 
+# We have assigned a series of learning rate values to create an optimized annealing schedule `lr_1, lr_2, lr_3` and corresponding number of epochs for each `epochs_1, epochs_2, epochs_3`. Feel free to change these values or you can just use the defaults we have filled in. 
```

Comment:
Hmm, sorry for not being clear but what I wanted to say is that defining variables which are just used to fill an array just adds a burden to the code 😅 .  For instance if I want to add another value to the array and follow this pattern I would have to add `lr_4` and `epochs_4` then add them to the array (where I could just add them to the array). It would be much easier to directly have the arrays (such as `epochs_arr = [0.001, 0.0001, 0.00001]`) and explain in the comment why we are creating such arrays. These kind of behaviours can often lead to what we call [edge cases](https://en.wikipedia.org/wiki/Edge_case) in the dev world.